easybuild/easyconfigs/c/cDNA_Cupcake/cDNA_Cupcake-24.2.0-foss-2020a-Python-3.8.2.eb

```
@@ -0,0 +1,50 @@
+# Contribution by
+# DeepThought, Flinders University
+# Updated to v24.2.0
+# R.QIAO <rob.qiao@flinders.edu.au>
+
+easyblock = 'PythonPackage'
+
+name = 'cDNA_Cupcake'
+version = '24.2.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/Magdoll/cDNA_Cupcake'
+description = """cDNA_Cupcake is a miscellaneous collection of Python and
+R scripts used for analyzing sequencing data.
+"""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+
+source_urls = ['https://github.com/Magdoll/cDNA_Cupcake/archive/refs/tags']
+sources = ['v%(version)s.tar.gz']
+checksums = ['9aa75ef3d00e9983772021dca3c5595df5ea23c39fb0004e4330e55edc782002']
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('SciPy-bundle', '2020.03', versionsuffix),
+    ('Biopython', '1.78', versionsuffix),
+    ('bx-python', '0.8.9', versionsuffix),
+    ('bcbio-gff', '0.6.6', versionsuffix),
+    ('scikit-learn', '0.23.1', versionsuffix),
+    ('Pysam', '0.16.0.1'),
+]
+
+download_dep_fail = False
+use_pip = True
```

@manifestoso we should always try to use download_dep_fail = True
the installation did not work for me because it has a wrong requirement on sklearn, which should be scikit-learn
this can be fixed as shown below:

```suggestion
download_dep_fail = True
use_pip = True

preinstallopts = "sed -i 's/sklearn/scikit-learn/g' setup.py && "
```

---

@smoors Thx for the spot 👍